charts/taraxa-node/templates/initconfig-consensus-node.yaml

```diff
@@ -0,0 +1,98 @@
+        {{- end }}
+      else
+        echo "Found wallet file."
+        KEY=$(cat "$WALLET_PATH" | jq -r .node_secret)
+        VRF=$(cat "$WALLET_PATH" | jq -r .vrf_secret)
+      fi
+    fi
+
+    {{ if .Values.explorer.enabled }}
+    {{- if and (ne .Values.config.network "1") (ne .Values.config.network "2") (ne .Values.config.network "3") }}
+    NODE_ADDRESS=$(taraxad --command account ${KEY} | grep node_address | cut -d\  -f3 | tr -d \")
+    curl --silent http://{{ .Release.Name }}-explorer/api/faucet/0x${NODE_ADDRESS}
+    {{- end }}
+    {{- end }}
+  
+    taraxad --command config \
+      --network-id {{ .Values.config.network }} \
+      --node-secret ${KEY} \
+      --vrf-secret ${VRF} \
+      {{- if and (ne .Values.config.network "1") (ne .Values.config.network "2") (ne .Values.config.network "3") }}
+      {{- $lbEnabled := .Values.bootnode.loadBalancer.enabled -}}
+      {{- $lbIPs := .Values.bootnode.loadBalancer.addresses -}}
+      {{- range $key, $value := .Values.config.bootnode.addresses }}
+      --boot-nodes {{ if $lbEnabled }}{{ index $lbIPs $key }}{{- else }}{{ include "taraxa-boot-node.fullname" $ }}-{{ $key }}.{{ include "taraxa-boot-node.fullname" $ }}.{{$.Release.Namespace}}{{- end }}:10002/{{ $value }} \
+      {{- end }}
+      {{- end }}
+      {{- if .Values.config.extraArgs }}
+      {{ join " " .Values.config.extraArgs }} \
+      {{- end }}
+      --config $CONFIG_PATH \
+      --wallet $WALLET_PATH \
+      --data-dir $DATA_PATH
+
+    {{- if and (ne .Values.config.network "1") (ne .Values.config.network "2") (ne .Values.config.network "3") }}
+    export DELEGATION_ADDRESS=$(cat $CONFIG_PATH | jq -r '.chain_config.final_chain.state.dpos.genesis_state | keys[0]')
+    export FIRST_NODE_ADDRESS=$(cat $CONFIG_PATH | jq -r '.chain_config.final_chain.state.dpos.genesis_state[env.DELEGATION_ADDRESS] | keys[0]')
+    export DELEGATION_VALUE=$(cat $CONFIG_PATH | jq -r '.chain_config.final_chain.state.dpos.genesis_state[env.DELEGATION_ADDRESS][env.FIRST_NODE_ADDRESS]')
+    cat $CONFIG_PATH | jq '.chain_config.final_chain.state.dpos.genesis_state[env.DELEGATION_ADDRESS] = ({})' > $CONFIG_PATH.tmp && mv $CONFIG_PATH.tmp $CONFIG_PATH
+    {{- range $key, $value := .Values.config.consensusnode.keys }}
+    echo "Adding node {{ $key }} address to the config..."
+    export ADDRESS="0x$(taraxad --command account {{ $value }} | grep "node_address" | awk '{print $3}' | tr -d '"')"
+    cat $CONFIG_PATH | jq '.chain_config.final_chain.state.dpos.genesis_state[env.DELEGATION_ADDRESS] += ({(env.ADDRESS): env.DELEGATION_VALUE})' > $CONFIG_PATH.tmp && mv $CONFIG_PATH.tmp $CONFIG_PATH
+    {{- end }}
+    {{- end }}
+
+    {{- if and (ne .Values.config.network "1") (ne .Values.config.network "2") (ne .Values.config.network "3") }}
+    {{ if .Values.explorer.enabled }}
+    export FAUCET_ADDRESS=$(taraxad --command account {EXPLORER_FAUCET_PRIVATE_KEY} | grep node_address | cut -d\  -f3 | tr -d \")
```

Comment:
If i understood correctly, this should be
`${EXPLORER_FAUCET_PRIVATE_KEY}`

please double check variable expansion

Reply:
I want to leave that as it is now. I dont want to make to many changes / steps at the same time.

`testnet` and `mainnet` and `prnet` are done in an old way from old repo.

`devnet` will be done:
- from new repos (taraxa-node and explorer)
- gitops deployment

> We should change the test/run
> 
> https://github.com/Taraxa-project/taraxa-node/blob/cf46cbd7e17125f6f63b5d361cebed4e783ad277/.circleci/config.yml#L580-L602
> 
> Changing the default values and moving to this repo will make this run deprecated, as we will use an old version of the chart.
> 
> So we should change the circleCI to use the local chart, and somehow make it use a values files for PR tests or leave defaults for it

Reply:
ok, so we are not touching the circleci tests.